<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stick Arena</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Not connected</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTextBrowser" name="textBrowser_2">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="styleSheet">
               <string notr="true">background-color: #202A31</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_mainChat"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTableWidget" name="userTable">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Username</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>R</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>G</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>B</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>96</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Player count:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_upTime">
                <property name="maximumSize">
                 <size>
                  <width>96</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Uptime: </string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Private</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTextBrowser" name="textBrowser_PM"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="lineEdit_PMName">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maxLength">
               <number>20</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_PM">
              <property name="maxLength">
               <number>135</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Account</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="changeAccount">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Change account</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="action2_Dimensional_Center"/>
    <addaction name="actionPaper_Thin_City"/>
    <addaction name="actionFine_Line_Island"/>
    <addaction name="actionU_of_SA"/>
    <addaction name="actionFlat_World"/>
    <addaction name="actionPlanar_Outpost"/>
    <addaction name="actionMobius_Metropolis"/>
    <addaction name="actionAmsterdam"/>
    <addaction name="actionSticktopia"/>
    <addaction name="actionCartesian_Republic"/>
   </widget>
   <addaction name="menuServer"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action2_Dimensional_Center">
   <property name="text">
    <string>2-Dimensional Central</string>
   </property>
  </action>
  <action name="actionPaper_Thin_City">
   <property name="text">
    <string>Paper Thin City</string>
   </property>
  </action>
  <action name="actionFine_Line_Island">
   <property name="text">
    <string>Fine Line Island</string>
   </property>
  </action>
  <action name="actionU_of_SA">
   <property name="text">
    <string>U of SA</string>
   </property>
  </action>
  <action name="actionFlat_World">
   <property name="text">
    <string>Flat World</string>
   </property>
  </action>
  <action name="actionPlanar_Outpost">
   <property name="text">
    <string>Planar Outpost</string>
   </property>
  </action>
  <action name="actionMobius_Metropolis">
   <property name="text">
    <string>Mobius Metropolis</string>
   </property>
  </action>
  <action name="actionAmsterdam">
   <property name="text">
    <string>Amsterdam</string>
   </property>
  </action>
  <action name="actionSticktopia">
   <property name="text">
    <string>Sticktopia</string>
   </property>
  </action>
  <action name="actionCartesian_Republic">
   <property name="text">
    <string>Cartesian Republic</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
